### DISCUSSION TITLE
Creation of a +/ - range slider

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [simeydotme/svelte-range-slider-pips#134] [Formatting / Styling] [answered]

**Comment by simeydotme** [comment]
Hi @MuslemRahimi  ... this is not a built-in feature (positive-negative range) ... but it is fun, i've never seen it before.

it's totally possible with a little bit of CSS and custom events. I can do a demo for you later if you're OK with that. If you already have a solution, let me know.

**Comment by simeydotme** [comment, marked as answer]
https://svelte.dev/repl/8947a3d09c3d442fa55d1db58093c6a7?version=4.2.11
^^ here you go. Edit as you need

**Reply by simeydotme** [reply]
you gotta use your brain a little , dude @_@

```diff
- --range: {Math.abs(values[0])}%
+ --range: {Math.abs(values[0]/2)}%
```

you doubled the range, so you needed to halve the %

**Reply by simeydotme** [reply]
could also do `values[0] / ((max - min)/100)` 

100 - -100 = 200
200 / 100 = 2

**Comment by simeydotme** [comment]
I've added this as a demo recipe@ https://simeydotme.github.io/svelte-range-slider-pips/en/recipes/positive-negative
